koku/reporting/migrations/0260_standard_numeric_on_calculated_feilds.py

```diff
@@ -0,0 +1,1403 @@
+# Generated by Django 3.2.15 on 2022-10-05 15:40
```

Comment:
This migration will have to be re-written. We'll need the column updates consolidated into a single SQL query per table. Else this will consume all of the head space in the DB.

We did a migration similar to this a while back and we had to change the migration because it wrecked production.

Reply:
Oh good to know. I can write it up as single query per table. 👍